astronomer/providers/core/example_dags/example_file_sensor.py

```diff
@@ -24,10 +26,30 @@
     default_args=default_args,
     tags=["example", "async", "core"],
 ) as dag:
+    start = EmptyOperator(task_id="start")
+
+    create_file = BashOperator(
+        task_id="create_file",
+        bash_command="sleep 10 && touch /usr/local/airflow/dags/example_file_async_sensor.txt",
```

Comment:
Since we're creating and sensing this file, perhaps we don't need to create it in the DAGs directory. How about creating it in the /tmp/ directory?
```suggestion
        bash_command="sleep 10 && touch /tmp/example_file_async_sensor.txt",
```

Reply:
This file needs to be accessable from the trigger since we are checking file status from the trigger.
so if I'll create a file with bash operator in /tmp/ folder then it would be available in the worker /tmp/ folder which trigger can't access but if I'll create in dags then it can access  since dags folder is a shared folder

Reply:
Is that only the DAGs folder is synced across Airflow components or the whole directory /usr/local/airflow synced? Maybe we can keep it there then. Only concern is that the dag processor would try to process this file and then find that it is not a DAG; not sure if we need to worry for that optimisation, but just a thought.

@kaxil do you have some inputs here?

Reply:
In future maybe either we can add `AIRFLOW__CORE__DAG_IGNORE_FILE_SYNTAX` in dockerfile or we can create a `. airflowignore`

Reply:
tracking here https://github.com/astronomer/astronomer-providers/issues/491

Reply:
Any file apart from `.py` files will be ignored when parsing for DAG files so not a problem